docs/scripts/generate-termynal.py

```diff
@@ -0,0 +1,148 @@
+                except StopIteration:
+                    pass
+
+            elif index == 1:  # The subprocess has exited.
+                output = child.before
+                interaction_history += [line.strip() for line in output.splitlines()]
+                break
+            elif index == 2:  # Timeout waiting for new data.
+                print("\nTimeout waiting for subprocess response.")
+                continue
+
+    finally:
+        return interaction_history
+
+
+ccds_script = [
+    ("project_name", "My Analysis"),
+    ("repo_name", "my_analysis"),
+    ("module_name", ""),
+    ("author_name", "Dat A. Scientist"),
+    ("description", "This is my analysis of the data."),
+    ("python_version_number", "3.12"),
+    ("Choose from", "3"),
+    ("bucket", "s3://my-aws-bucket"),
+    ("aws_profile", ""),
+    ("Choose from", "2"),
+    ("Choose from", "1"),
+    ("Choose from", "2"),
+    ("Choose from", "2"),
+    ("Choose from", "1"),
+]
+
+
+def run_scripts():
+    try:
+        output = []
+        output += execute_command_and_get_output(f"ccds {CCDS_ROOT}", ccds_script)
+        return output
+
+    finally:
+        # always cleanup
+        if Path("my_analysis").exists():
+            shutil.rmtree("my_analysis")
+
+
+def render_termynal():
+    # actually execute the scripts and capture the output
+    results = run_scripts()
+
+    # watch for inputs and format them differently
+    script = iter(ccds_script)
+    _, user_input = next(script)
+
+    conv = Ansi2HTMLConverter(inline=True)
+    html_lines = [
+        '<div id="termynal" data-termynal class="termy" data-ty-macos data-ty-lineDelay="100" data-ty-typeDelay="50" title="Cookiecutter Data Science">'
+    ]
+    result_collector = []
+
+    for line_ix, result in enumerate(results):
+        # style bash user inputs
+        if result.startswith("$"):
+            result = conv.convert(result.strip("$"), full=False)
+            html_lines.append(
+                f'<span data-ty="input" data-ty-prompt="$">{result}</span>'
+            )
+
+        # style inline cookiecutter user inputs
+        elif ":" in result and user_input in result:
+            # treat all the options that were output as a single block
+            if len(result_collector) > 1:
+                prev_results = conv.convert(
+                    "\n".join(result_collector[:-1]), full=False
+                )
+                html_lines.append(f"<span data-ty>{prev_results}</span>")
+
+            # split the line up into the prompt text with options, the default, and the user input
+            prompt, user_input = result.strip().split(":", 1)
+            prompt = conv.convert(prompt, full=False)
+            prompt = f'<span data-ty class="inline-input">{result_collector[-1].strip()} {prompt}:</span>'
+            user_input = conv.convert(user_input.strip(), full=False)
+
+            # treat the cookiecutter prompt as a shell prompt
+            out_line = f"{prompt}"
```

Comment:
Sorry, I missed a _very_ important line in the suggestion
```suggestion
            out_line = f"{prompt}"
            out_line += f'<span class="inline-input" data-ty="input" data-ty-delay="500" data-ty-prompt="">{user_input}</span>'
```